pkgs/tools/misc/bandwidth/default.nix

```diff
@@ -19,7 +19,8 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ nasm ];
 
-  buildFlags = [ arch ];
+  buildFlags = [ arch ]
+    ++ stdenv.lib.optionals stdenv.cc.isClang [ "CC=clang" "LD=clang" ];
```

Comment:
I think stdenv has no way of knowing that CC and LD are valid make flags - they're a convention, not a rule.

The reason this comes up so often is that Apple choose to symlink gcc and g++ to clang. Consequently, people kept using gcc/g++ in their makefiles. But clang is not gcc, it behaves differently, so I can understand why we chose not to emulate this behavior.